src/GraphPicker.jsx

```diff
@@ -18,7 +18,7 @@ export default class GraphPicker extends React.Component {
                 title={displayGraphType(n)}
                 onClick={() => this.props.onSelectGraph(n)}>
                 <div className="card-img-top">
-                    <img className="img-fluid" src={this.mediaPrefix + imgname} />
+                    <img className="img-fluid" alt="" src={this.mediaPrefix + imgname} />
```

Comment:
The empty alt property indicates an image is decorative and should not be focused on. Since the title of the graph is indicated on the card along with an image description would be redundant.